### PR TITLE
belt naming

### PR DESCRIPTION
**Pull request by Casper3667**
The ordinary belt in the loadout can now be renamed & given a new description.

### CODE REVIEW

**Comment by Casper3667** [comment]
!review

**Review by Geevies — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_belt.dm [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
 	slot = slot_belt
 	sort_category = "Belts"
-	flags = GEAR_HAS_COLOR_SELECTION
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
 /datum/gear/belt/fannypack
 	display_name = "fannypack"
 	cost = 2
 	path = /obj/item/storage/belt/fannypack/recolorable
+	flags = GEAR_HAS_COLOR_SELECTION
```
Comment (Alberyk) [inline code/modules/client/preference_setup/loadout/loadout_belt.dm:12]: That will make so that you can't rename the fanny pack.

**Review by Alberyk — CHANGES_REQUESTED** [review]

File: code/modules/client/preference_setup/loadout/loadout_belt.dm (hunk 1 above)
Comment (Casper3667) [reply code/modules/client/preference_setup/loadout/loadout_belt.dm:12]: That was the intention as I didn't intend to make changes to how the fanny pack currently is (currently it only allows color selection), only to the basic belt.

**Review by Alberyk — APPROVED** [review]

**Review by JohnWildkins — APPROVED** [review]
honestly letting fannypacks be named too is fine

**Pull request merged**